test/wasmBenchmarker/ctests/simdMandelbrotCount.c

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2023-present Samsung Electronics Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdio.h>
+#include <stdbool.h>
+#include <stdint.h>
+
+#include <immintrin.h> // SIMD header
+
+#define WIDTH 1600
+#define HIGHT 1400
+#define N 20
+#define REAL_AXIS_SHIFT -1.8 // ~ horizontal shift
+#define IMAGINARY_AXIS_SHIFT -1.0 // ~ vertical shift
+#define ZOOM 0.0015
+
+#define getNthBit(b, n) ((b & (0b00000001 << (7 - n))) > 0)
+
+#define setNthBitZero(b, n) b = b & (0b11111111 - (0b00000001 << (7 - n)))
+
+#define setNthBitOne(b, n) b = b | (0b00000001 << (7 - n))
+
+#define SQUARE(num) _mm_mul_ps(num, num)
+
+#define ABS_COMPLEX(z_real, z_complex) _mm_sqrt_ps(_mm_add_ps(SQUARE(z_real), SQUARE(z_imaginary)))
+
+typedef uint8_t byte;
+
+byte areInMandelbrotSet(__m128 c_real, __m128 c_imaginary);
+
+uint32_t runtime();
+
+int main() {
```

Comment:
Man could be moved below, and runtime forward definition is not needed.

Reply:
Okay.